orttraining/orttraining/python/training/ortmodule/_runtime_inspector.py

```diff
@@ -21,8 +19,7 @@ def __init__(self):
 class InputDensityObserver:
     """Configurable input data observer for ORTModule."""
 
-    def __init__(self, log_steps=10):
-        self._enabled = ortmodule._defined_from_envvar("ORTMODULE_ENABLE_INPUT_DENSITY_INSPECTOR", 0, warn=True) == 1
```

Comment:
can we keep the option to disable? We can simply turn it ON by default...

Reply:
I leave `ORTMODULE_ENABLE_INPUT_DENSITY_INSPECTOR` to be there, but in a different place to control `InputDensityObserver` instance's creation (if enabled). 
So that means, the switch is outside of InputDensityObserver's implementation now.

Reply:
in the latest commit, ORTMODULE_ENABLE_LABEL_SPARSITY_OPT is used to turn on or off the feature.